README.md

```diff
@@ -93,20 +93,43 @@ This test continuously applies virtual services and sends traffic to see how lon
 Usage: `pilot-load prober --replicas=1000 --delay=1s`.
 
 ## Reproduce
+The `reproduce` command allows replaying a cluster's configuration. Install `kubectl grep`
+[plugin](https://github.com/howardjohn/kubectl-grep)  before running below command.
 
-The `reproduce` command allows replaying a cluster's configuration.
+First, capture their current cluster config: 
 
-First, capture their current cluster config: `kubectl get vs,gw,dr,sidecar,svc,endpoints,pod,namespace -oyaml -A | kubectl grep`
+`kubectl get authorizationpolicies,destinationrules,envoyfilters,gateways,peerauthentications,requestauthentications,serviceentries,sidecars,telemetries,virtualservices,configmaps,svc,endpoints,pod,namespace -oyaml -A | kubectl grep > my-config.yaml`
```

Comment:
```suggestion
`kubectl get istio-io,configmaps,svc,endpoints,pod,namespace -oyaml -A | kubectl grep > my-config.yaml`
```

gets all the Istio ones. I assume we want all of them?

Reply:
We are not getting IstioOperator as it is more for customising Istio Installation. We are also not getting WorkloadEntry and WorkloadGroup.

Reply:
Taking WE and WG, Thank you.